src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -659,9 +659,9 @@ private string GetDefaultDurableV2HubName()
         internal HttpRequestMessage BuildSetTriggersRequest()
         {
             var url = default(string);
-            if (_environment.IsKubernetesManagedHosting())
+            if (_environment.IsKubernetesManagedHosting() || _environment.IsManagedAppEnvironment())
```

Comment:
In the API server PR, there was discussion on https support, and I believe it was added. Should you use https here?

Reply:
Env var `FUNCTIONS_API_SERVER` is already set with the https protocol used. Env var looks like this 
`FUNCTIONS_API_SERVER=https://<appname>.internal.<DNSsuffix>`.